### ISSUE TITLE
Moving death tiles with projectile collision

### ISSUE THREAD

**Issue by duralakun** [necropotame/teeworlds-infclass#173]
Right now you can shoot trough moving death tiles. It would be nice if there was a second type of moving death that doesnt allow bullets/grenades/laser trough

**Comment by necropotame** [comment]
I agree. I could implement it without laser bouncing to start with something simple
